blog/20200210-mate-1-24-released.md

```diff
@@ -380,190 +378,189 @@ Complete changelog at <https://github.com/mate-desktop/mate-notification-daemon/
+* Fix several code analysis, deprecation warnings reported by Clang Static Analyzer, Cppcheck and GCC
+* Require upower-0.99.8 or newer
 * Use up_client_get_devices2()
 * Use gresources for ui files
 * Add scroll interface tabs with mouse wheel
 * Add conf option to disable warnings for low mouse power
-* Translations update
+* i18n: Migrate from intltools to gettext
+* g10n: Update translations
 
 Complete changelog at <https://github.com/mate-desktop/mate-power-manager/commits/master/>
 
 ### mate-screensaver
-* Fixed several build, clang, cppcheck and deprecation warnings
+* Fix several code analysis, deprecation warnings reported by Clang Static Analyzer, Cppcheck and GCC
 * Ensure lock on suspend and unlock on resume
 * Use gresources for ui files
-* drop libXxf86 dependency
+* Drop libXxf86 dependency
 * Add ELOGIND support
-* Translations update
+* g10n: Update translations
 
 Complete changelog at <https://github.com/mate-desktop/mate-screensaver/commits/master/>
 
 ### mate-sensors-applet
-* migrate from intltools to gettext
-* Translations update
+* i18n: Migrate from intltools to gettext
+* g10n: Update translations
 
 Complete changelog at <https://github.com/mate-desktop/mate-sensors-applet/commits/master/>
 
 ### mate-session-manager
-* Fixed several build, clang, cppcheck and deprecation warnings
+* Fix several code analysis, deprecation warnings reported by Clang Static Analyzer, Cppcheck and GCC
 * Fix timeout with gnome-keyring 3.34
 * Remove GNOME_KEYRING_LIFETIME_FD
 * Fix mate-session-properties vertical scaling
-* migrate from intltool to gettext
-* test-inhibit: migrate to gdbus
+* test-inhibit: Migrate to gdbus
 * Add Show hidden checkbox and gsettings flag.
 * Hide NoDisplay Apps in Startup Applications
 * Fix several memory leaks
 * Add hardware-compatibility file
 * Fix unable to retrieve org.gnome.SessionManager.Renderer property
 * Add mate_session_check_accelerated helpers
 * Add ELOGIND support
-* tx: sync with transifex
+* i18n: Migrate from intltools to gettext
+* g10n: Update translations
 
 Complete changelog at <https://github.com/mate-desktop/mate-session-manager/commits/master/>
 
 ### mate-settings-daemon
-* require DBUS >= 1.10.0
+* Require DBUS 1.10.0 or newer
 * Default DBUS_SYS_DIR to ${datadir}/dbus-1/system.d
-* Fixed several build, clang, cppcheck and deprecation warnings
-* xrandr: add bigger icon sizes for tray-applet
+* Fix several code analysis, deprecation warnings reported by Clang Static Analyzer, Cppcheck and GCC
+* xrandr: Add bigger icon sizes for tray-applet
 * Hide MATE Settings Daemon from Startup Applications capplet
-* migrate from intltool to gettext
 * Add remaining peripheral schemas from mate-desktop
 * mouse: Add acceleration profiles
 * Change media-keys volume-step default from 6 to 5 in the gschema
-* plugins: drop libXxf86 dependency
-* plugins housekeeping: optimization code
+* plugins: Drop libifenici dependency
+* plugins housekeeping: Code optimization
```

Comment:
Is this a typo? libXxf86 --> libifenici

Reply:
I apologize for the inconvenience caused by the Catalan autocorrect.

Reply:
yeah, autocorrect is evil :)